src/lib/tradingView/dydxfeed/index.ts

```diff
@@ -92,11 +92,10 @@ export const getDydxDatafeed = (
     const pricescale = tickSizeDecimals ? 10 ** tickSizeDecimals : initialPriceScale ?? 100;
 
     const symbolInfo: LibrarySymbolInfo = {
-      ticker: symbolItem.full_name,
-      name: symbolItem.symbol,
+      ticker: symbolItem.symbol,
```

Comment:
`LibrarySymbolInfo.full_name` was deprecated

we're setting `market.id` as both the `symbol` and `name` right now, so these values are interchangeable. im setting `full_name` on `name` instead because `LibrarySymbolInfo` is always defined whereas `LibrarySymbolInfo.ticker` is ` | undefined`